pkg/api/handlers/libpod/pods.go

```diff
@@ -248,6 +249,21 @@ func PodDelete(w http.ResponseWriter, r *http.Request) {
 	}
 	ctrs, err := runtime.RemovePod(r.Context(), pod, true, query.Force, query.Timeout)
 	if err != nil {
+		if len(ctrs) > 0 {
+			// We have container errors to send as well.
+			// Since we're just writing an error, and we don't want
+			// special error-handling for just this endpoint: use a
+			// multierror to package up all container errors.
+			var allCtrErrors error
+			for _, ctrErr := range ctrs {
+				allCtrErrors = multierror.Append(allCtrErrors, ctrErr)
+			}
+
+			// This loses error typing, but since it's going to be
+			// JSON text once it hits the wire, it doesn't matter.
+			err = fmt.Errorf("%s. %s", err.Error(), allCtrErrors.Error())
```

Comment:
Alternatively, you could use `pkg/errorhandling.JoinErrors(...)`.

Reply:
That'll nest a second multierror on top of the first one, I don't think that's a good idea, formatting wise